x/concentrated-liquidity/pool_test.go

```diff
@@ -1,94 +1,100 @@
 package concentrated_liquidity_test
 
 import (
-	fmt "fmt"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 func (s *KeeperTestSuite) TestCalcOutAmtGivenIn() {
 	ctx := s.Ctx
-	pool, err := s.App.ConcentratedLiquidityKeeper.CreateNewConcentratedLiquidityPool(s.Ctx, 1, "eth", "usdc", sdk.NewInt(0), sdk.NewInt(0))
+	pool, err := s.App.ConcentratedLiquidityKeeper.CreateNewConcentratedLiquidityPool(ctx, 1, "eth", "usdc", sdk.MustNewDecFromStr("70.710678"), sdk.NewInt(85176))
 	s.Require().NoError(err)
 
 	// test asset a to b logic
 	tokenIn := sdk.NewCoin("eth", sdk.NewInt(133700))
 	tokenOutDenom := "usdc"
 	swapFee := sdk.NewDec(0)
+	minPrice := sdk.NewDec(4500)
+	maxPrice := sdk.NewDec(5500)
 
-	amountOut, err := pool.CalcOutAmtGivenIn(ctx, tokenIn, tokenOutDenom, swapFee)
+	_, amountOut, err := s.App.ConcentratedLiquidityKeeper.CalcOutAmtGivenIn(ctx, tokenIn, tokenOutDenom, swapFee, minPrice, maxPrice, pool.Id)
 	s.Require().NoError(err)
-	s.Require().Equal(sdk.NewDec(663944647).String(), amountOut.Amount.ToDec().String())
+	s.Require().Equal(sdk.NewDec(666975610).String(), amountOut.Amount.ToDec().String())
 
 	// test asset b to a logic
 	tokenIn = sdk.NewCoin("usdc", sdk.NewInt(4199999999))
 	tokenOutDenom = "eth"
 	swapFee = sdk.NewDec(0)
 
-	amountOut, err = pool.CalcOutAmtGivenIn(ctx, tokenIn, tokenOutDenom, swapFee)
+	_, amountOut, err = s.App.ConcentratedLiquidityKeeper.CalcOutAmtGivenIn(ctx, tokenIn, tokenOutDenom, swapFee, minPrice, maxPrice, pool.Id)
 	s.Require().NoError(err)
 	s.Require().Equal(sdk.NewDec(805287), amountOut.Amount.ToDec())
 
+	// test asset b to a logic
+	tokenIn = sdk.NewCoin("usdc", sdk.NewInt(42000000))
+	tokenOutDenom = "eth"
+	swapFee = sdk.NewDec(0)
+
+	_, amountOut, err = s.App.ConcentratedLiquidityKeeper.CalcOutAmtGivenIn(ctx, tokenIn, tokenOutDenom, swapFee, minPrice, maxPrice, pool.Id)
```

Comment:
utack on these test cases, I would love to see detailed comments on where these numbers are from or how the math is done to get these numbers, don't think it has to be in the scope of this PR